filament/include/filament/Frustum.h

```diff
@@ -50,24 +53,55 @@ class UTILS_PUBLIC Frustum {
     Frustum& operator=(const Frustum& rhs) = default;
     Frustum& operator=(Frustum&& rhs) noexcept = default;
 
-    // create a frustum from a projection matrix (usually the projection * view matrix)
+    /**
+     * Create a frustum from a projection matrix (usually the projection * view matrix)
```

Comment:
We usually use descriptive voice (Creates, Sets, Returns) rather than imperative (Create, Set, Return)